subscriptions/views.py

```diff
@@ -102,4 +103,28 @@ def get(self, request):
                 }for review in subscription.review_set.all()]
         }for subscription in subscription]
         
-        return JsonResponse({'results': results}, status=200)
\ No newline at end of file
+        return JsonResponse({'results': results}, status=200)
+
+class Reviewlistview(View) :
+    def get(self, request) :
+        reviews       = Review.objects.all()
+        reviews_count = reviews.count()
```

Comment:
ORM 최적화 시간에 배운 내용을 적용해 보면 전체 카운트를 세는 것은 ORM을 이용해 count를 하는 것보다. len()을 이용는 것이 더 효율적입니다.
```suggestion
        reviews_count = len(reviews)
```

Reply:
수정했습니다!